ERROR - 2023-08-29 13:01:35 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in E:\xampp\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\xampp\\htdocs...', 173)
#1 E:\xampp\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('https://www.ove...', 'overninja', Object(SensitiveParameterValue), 'overninja_db', 3306, '', 0)
#2 E:\xampp\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\xampp\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\xampp\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 E:\xampp\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 E:\xampp\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 E:\xampp\htdocs\app\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
#8 E:\xampp\htdocs\system\CodeIgniter.php(932): App\Controllers\Home->test()
#9 E:\xampp\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 E:\xampp\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in E:\xampp\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\xampp\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('https://www.ove...', 'overninja', Object(SensitiveParameterValue), 'overninja_db', 3306, '', 0)
#1 E:\xampp\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\xampp\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\xampp\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\xampp\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 E:\xampp\htdocs\app\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
#7 E:\xampp\htdocs\system\CodeIgniter.php(932): App\Controllers\Home->test()
#8 E:\xampp\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in E:\xampp\htdocs\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\xampp\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\xampp\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\xampp\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\xampp\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 E:\xampp\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 E:\xampp\htdocs\app\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
#6 E:\xampp\htdocs\system\CodeIgniter.php(932): App\Controllers\Home->test()
#7 E:\xampp\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 E:\xampp\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-29 13:01:35 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `contacts`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->test()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-29 13:01:47 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in E:\xampp\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\xampp\\htdocs...', 173)
#1 E:\xampp\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('https://www.ove...', 'overninja', Object(SensitiveParameterValue), 'overninja_db', 3306, '', 0)
#2 E:\xampp\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\xampp\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\xampp\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 E:\xampp\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 E:\xampp\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 E:\xampp\htdocs\app\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
#8 E:\xampp\htdocs\system\CodeIgniter.php(932): App\Controllers\Home->test()
#9 E:\xampp\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 E:\xampp\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in E:\xampp\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\xampp\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('https://www.ove...', 'overninja', Object(SensitiveParameterValue), 'overninja_db', 3306, '', 0)
#1 E:\xampp\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\xampp\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\xampp\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\xampp\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 E:\xampp\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 E:\xampp\htdocs\app\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
#7 E:\xampp\htdocs\system\CodeIgniter.php(932): App\Controllers\Home->test()
#8 E:\xampp\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 E:\xampp\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in E:\xampp\htdocs\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\xampp\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\xampp\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\xampp\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\xampp\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 E:\xampp\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 E:\xampp\htdocs\app\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
#6 E:\xampp\htdocs\system\CodeIgniter.php(932): App\Controllers\Home->test()
#7 E:\xampp\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 E:\xampp\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-29 13:01:47 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `contacts`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(98): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->test()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-29 13:23:38 --> App\Controllers\Home::test(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 99.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->test()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-29 13:25:30 --> App\Controllers\Home::test(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 99.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->test()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-29 17:32:03 --> App\Controllers\Home::test(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 122.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Home->test()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
